packages/management/addons-manager/bundle/config/upstream/addons-manager.yaml

```diff
@@ -0,0 +1,179 @@
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  labels:
+    app: tanzu-addons-manager
+  name: tanzu-addons-manager-sa
+  namespace: tanzu-system
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: tanzu-addons-manager-clusterrole
+rules:
+- apiGroups:
+  - run.tanzu.vmware.com
+  resources:
+  - tanzukubernetesreleases
+  - tanzukubernetesreleases/status
+  verbs:
+  - get
+  - list
+  - watch
+- apiGroups:
+  - cluster.x-k8s.io
+  resources:
+  - clusters
+  - clusters/status
+  verbs:
+  - get
+  - list
+  - watch
+- apiGroups:
+  - controlplane.cluster.x-k8s.io
+  resources:
+  - kubeadmcontrolplanes
+  - kubeadmcontrolplanes/status
+  verbs:
+  - get
+  - list
+  - watch
+- apiGroups:
+  - ""
+  resources:
+  - secrets
+  verbs:
+  - get
+  - list
+  - watch
+  - create
+  - update
+  - patch
+  - delete
+- apiGroups:
+  - ""
+  resources:
+  - configmaps
+  verbs:
+  - get
+  - list
+  - watch
+- apiGroups:
+  - ""
+  resources:
+  - events
+  verbs:
+  - create
+  - patch
+- apiGroups:
+  - kappctrl.k14s.io
+  resources:
+  - apps
+  verbs:
+  - get
+  - list
+  - watch
+  - create
+  - update
+  - patch
+  - delete
+- apiGroups:
+  - packaging.carvel.dev
+  resources:
+  - packagerepositories
+  - packagerepositories/status
+  verbs:
+  - get
+  - list
+  - watch
+  - create
+  - update
+  - patch
+- apiGroups:
+  - packaging.carvel.dev
+  resources:
+  - packageinstalls
+  - packageinstalls/status
+  verbs:
+  - get
+  - list
+  - watch
+  - create
+  - update
+  - patch
+  - delete
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRoleBinding
+metadata:
+  name: tanzu-addons-manager-clusterrolebinding
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: ClusterRole
+  name: tanzu-addons-manager-clusterrole
+subjects:
+- kind: ServiceAccount
+  name: tanzu-addons-manager-sa
+  namespace: tanzu-system
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  labels:
+    app: tanzu-addons-manager
+  name: tanzu-addons-controller-manager
+  namespace: tanzu-system
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: tanzu-addons-manager
+  template:
+    metadata:
+      labels:
+        app: tanzu-addons-manager
+    spec:
+      containers:
+      - args:
+        - --metrics-bind-addr=0
+        - --leader-elect=false
+        - --health-addr=:18316
+        image:  projects-stg.registry.vmware.com/tkg/tanzu_core_addons/tanzu-addons-manager:v1.5.0_vmware.1
```

Comment:
we should not be hardcoding the image URL here, we should instead rely on kbld config to override the image URL. Check https://github.com/vmware-tanzu/tanzu-framework/blob/main/packages/kbld-config.yaml and add an entry for addons-manager image here

Reply:
added addons-manager image in kbld-config.yaml